### PR TITLE
Initial commit - WireShare

### PR DESCRIPTION
**Pull request by nmatavka**
I tried doing a pull request against new-pr, but GitHub wouldn't let me do that, so I'm doing the closest thing I can.

This is the manifest for WireShare, a Java app with a tremendously long history, and one that has shaped Internet culture as experienced by the Millennial Generation.  You may have heard of LimeWire, the program that became synonymous with music "piracy" and ultimately succumbed to a relentless campaign waged in the media, the courts, and every other public forum by the RIAA.  WireShare is LimeWire all grown up, with its ambitions focussed no longer on copyright infringement, but on rapid public dissemination of large files.  

HERMES WireShare is one of only three Gnutella servents (server/clients) in active development, together with Shareaza and gtk-Gnutella.  It is also by far the most commonly used.

### CODE REVIEW

File: org.team_hermes.WireShare.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+id: org.team_hermes.WireShare
+branch: 6.0
+runtime: org.freedesktop.Platform
+runtime-version: 19.08
+sdk: org.freedesktop.Sdk
+command: /app/WireShare.sh
+modules:
```
Comment (bilelmoussaoui) [inline org.team_hermes.WireShare.yaml]: I think you need  org.freedesktop.Sdk.Extension.openjdk11 extension.

File: org.team_hermes.WireShare.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+id: org.team_hermes.WireShare
+branch: 6.0
+runtime: org.freedesktop.Platform
+runtime-version: 19.08
```
Comment (bilelmoussaoui) [inline org.team_hermes.WireShare.yaml]:
```suggestion
runtime-version: '19.08'
```
That's a string.

File: org.team_hermes.WireShare.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+id: org.team_hermes.WireShare
+branch: 6.0
```
Comment (bilelmoussaoui) [inline org.team_hermes.WireShare.yaml]:
```suggestion
```
Flathub doesn't use this at all.

File: org.team_hermes.WireShare.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    build-commands:
+      - install WireShare.sh /app/WireShare.sh
+      - install WireShare.jar /app/WireShare.jar
+      - mkdir -p /app/share/icons/
+      - cp -r hicolor /app/share/icons
+      - cp -r jre-linux /app/jre
+      - cp -r lib64 /app/lib
+      - mkdir -p /app/share
+      - cp -r applications /app/share/applications
+finish-args:
```
Comment (bilelmoussaoui) [inline org.team_hermes.WireShare.yaml]: Put the finish-args before the modules.

File: org.team_hermes.WireShare.yaml (hunk 1 above)
Comment (nmatavka) [reply org.team_hermes.WireShare.yaml]: I'm vendoring my own, somewhat lobotomised OpenJDK; we have found that it works quite a bit faster than the Oracle-supplied OpenJDK.

File: org.team_hermes.WireShare.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: /app/WireShare.sh
+modules:
+  - name: wireshare
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://nmatavka@bitbucket.org/nmatavka/hermes-wireshare-flatpak.git
```
Comment (TingPing) [inline org.team_hermes.WireShare.yaml]: This should either be in tree or a clearly official source.

Comment (barthalion) [reply org.team_hermes.WireShare.yaml]: Yeah, can it be moved under `team-hermes` bitbucket org?

File: org.team_hermes.WireShare.yaml [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        branch: master
```
Comment (barthalion) [inline org.team_hermes.WireShare.yaml]: It should use specific commit/tag for reproducibility.

**Comment by barthalion** [comment]
Have you forgot to push some commits? I see you marked comments as resolved but they weren't.

**Comment by nmatavka** [comment]
Can this get merged already?

**Comment by barthalion** [comment]
@nmatavka Can you address comments I left? You marked them as resolved but I still don't see any changes.

**Pull request closed without merging**